test/apiv2/python/rest_api/test_v2_0_0_system.py

```diff
@@ -16,7 +16,9 @@ def test_info(self):
         r = requests.get(self.podman_url + "/v1.40/info")
         self.assertEqual(r.status_code, 200, r.text)
         self.assertIsNotNone(r.content)
-        _ = r.json()
+        response = r.json()
+        self.assertIn("name=rootless", response["SecurityOptions"])
```

Comment:
Should add a check here on whether the test is running as non-root.

Reply:
done